storage/grpc_client.go

```diff
@@ -1471,6 +1472,12 @@ func (r *gRPCReader) reopenStream() (*storagepb.ReadObjectResponse, error) {
 
 func newGRPCWriter(c *grpcStorageClient, params *openWriterParams, r io.Reader) *gRPCWriter {
 	size := params.chunkSize
+
+	// Round up chunksize to nearest 256KiB
+	if size%googleapi.MinUploadChunkSize != 0 {
+		size += googleapi.MinUploadChunkSize - (size % googleapi.MinUploadChunkSize)
```

Comment:
And, the if statement is superfluous.

Reply:
The if statement is needed when a user provides an exact multiple of 256kib. Without the if statement, the chunksize would be `chunksize+256kib` instead of just `chunksize`.

I am inclined to keep the code as-is since it's copied directly from [here](https://github.com/googleapis/google-api-go-client/blob/main/googleapi/googleapi.go#L248-L250); might as well keep it consistent.